input/pagecontent/StructureDefinition-dk-core-organization-intro.md

```diff
@@ -9,6 +9,7 @@ Please notice, that an organization shall always have at least one identifier. S
 - *CVR identifier*: an official company identifier for Danish companies.
 - *Kommunal identifier*: the official identifier for a Danish municipality.
 - *Regional identifier*: the official organization identifier for a Danish region.
+- *Producent identifier*: the identifier used for the producer of laboratory result. 
```

Comment:
Should state who defines the codes. I don't actually know who to contact if a lab does not have an identifier assigned to it.

Reply:
@tmsMedcom undersøger